manifest.yaml

```diff
@@ -0,0 +1,6 @@
+iso:
+  uefi:
+    - oci:registry.opensuse.org/isv/rancher/elemental/teal53/15.4/rancher/elemental-live-grub2-efi-image/5.3
```

Comment:
this dont exist yet do they?

Reply:
yes they do https://build.opensuse.org/package/show/isv:Rancher:Elemental:Teal53/live-grub2

Reply:
ah just saw the packages! arm64 is failing thougth. Spec file is gonna be a pita, Im happy its not me doing them :D

Reply:
Another options could be to build the live/grub2 as containers as part of the `elemental-iso-build` script and somehow render the Dockerfile according to the arch... 🤔 This could work but turns `elemental-iso-build` script into a little monster (probably not that little...)